src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/ResourceQueriesQueryString.feature.disabled

```diff
@@ -0,0 +1,105 @@
+Feature: Query String handling for GET requests for Resource Queries
+
+  Background:
```

Comment:
Could you confirm if the Background section is required?

Reply:
It might be useful to remove all of the "given a school exists" statements below, and have a single Background section with "Given a school exists".

Reply:
Complete